Feb 2025 newsletter blog and 2025 Plans blog (#314)
ddev/ddev.com: src/content/blog/ddev-feb-2025-newsletter.md

---
title: "DDEV February 2025 Newsletter"
pubDate: 2025-02-06
#modifiedDate: 2025-02-06
summary: DDEV February 2025 Newsletter
author: Randy Fay
#featureImage:
#  src: /img/blog/2025/01/mtgarfield.jpg
#  alt: Mount Garfield over Palisade, Colorado, 2025-01-07
#  credit: View of Mount Garfield above Palisade, Colorado on my walk to work today, January 6, 2025
categories:
  - Community
---

**Happy February!**

> **DDEV v1.24.2** was [released](https://github.com/ddev/ddev/releases/tag/v1.24.2) with [MySQL 8.4 support](https://ddev.com/blog/database-improvements) and improved `ddev composer create`.

Reviewer: > improved `ddev composer create`

It says it here, but doesn't say how it was improved anywhere.

**DDEV Notes and News**

- The [DDEV Advisory Group](https://github.com/orgs/ddev/discussions/6853) met January 8. The primary topic of discussion was the [change in DDEV's funding](https://ddev.com/blog/platform-sh-ddev-funding-changes) and budgeting response to it. Our decision was to continue the arrangement with Stas as-is (bill hours at $50/hour, cap $4000/month). Randy will also bill at $50/hour with cap of $4000, but limited by always trying to leave $6000 in the bank.
- macOS users will be interested in the (not-DDEV-focused) [Docker Performance macOS 2025](https://www.paolomainardi.com/posts/docker-performance-macos-2025/).
- [The Open Source Pledge and DDEV: A Path to Sustainability](https://ddev.com/blog/open-source-pledge).

> - [MySQL 8.4 is now in DDEV v1.24.2](https://ddev.com/blog/database-improvements/).

Reviewer: Note for Composer:

```suggestion
- [MySQL 8.4 is now in DDEV v1.24.2](https://ddev.com/blog/database-improvements/).
- DDEV v1.24.2 has improved argument handling for `ddev composer create`, making it nearly compatible with `composer create-project`. Additionally, the `ddev composer create-project` alias has been added for improved usability.
```

Author: What a great thing, too!

> - The Gitpod that we have known and loved is shutting down. This affects DDEV contributors that have used it and makes [Drupalpod as it stands obsolete](https://www.drupal.org/project/drupalpod/issues/3500792). If you're interested in Drupalpod and will be at DrupalCon Atlanta, there is a Birds-of-a-Feather session scheduled on the path forward, 04:00pm - 04:30pm Tuesday, March 25, 2025.

Reviewer: It probably doesn't matter anymore, but anyway:

```suggestion
- The Gitpod that we have known and loved is shutting down. This affects DDEV contributors that have used it and makes [DrupalPod as it stands obsolete](https://www.drupal.org/project/drupalpod/issues/3500792). If you're interested in DrupalPod and will be at DrupalCon Atlanta, there is a Birds-of-a-Feather session scheduled on the path forward, 04:00pm - 04:30pm Tuesday, March 25, 2025.
```

- The Drop Times did a nice and extensive [interview with Randy](https://www.thedroptimes.com/interview/45389/making-ddev-past-present-and-future-in-focus) and [video interview](https://www.youtube.com/watch?v=EctDtSZcBhU).
- Tag1 Consulting did a performance comparison of the new Drupal CMS vs WordPress, (using DDEV for everything). See [Drupal CMS vs WordPress Performance](https://www.tag1consulting.com/blog/drupal-cms-vs-wordpress-performance-2025).
- The latest DDEV Quickstarts in the docs are for [Drupal CMS](https://ddev.readthedocs.io/en/stable/users/quickstart/#drupal-drupal-cms) and [ProcessWire](https://ddev.readthedocs.io/en/latest/users/quickstart/#processwire)! Congratulations to the Drupal community on Drupal CMS, and welcome ProcessWire folks. Thanks to [Bernhard Baumrock](https://github.com/BernhardBaumrock) for the ProcessWire initiative!

**Ongoing DDEV Work**

> - **Node.js** improved support: People have asked for some time for Node.js to be the primary webserver (instead of Nginx+php-fpm, etc), and now it can be done in a number of ways. Your review of nearly-complete [Generic Webserver Support](https://github.com/ddev/ddev/pull/6935) is welcome. Incidentally, this also provides experimental support for [FrankenPHP](https://frankenphp.dev/) and provides a Quickstart for it.

Reviewer: ```suggestion
- **Node.js** improved support: People have asked for some time for Node.js to be the primary web server (instead of Nginx+php-fpm, etc), and now it can be done in a number of ways. Your review of nearly-complete [Generic Web Server Support](https://github.com/ddev/ddev/pull/6935) is welcome. Incidentally, this also provides experimental support for [FrankenPHP](https://frankenphp.dev/) and provides a Quickstart for it.
```
